person.rb

```diff
@@ -1,6 +1,6 @@
 require './nameable'
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals

```

Comment:
- [ ] Please set these variables to the attribute accessor. The reason is we might want to read and write. Attribute won't give us that functionality.